addon/components/course-rollover.js

```diff
@@ -46,6 +45,9 @@ export default class CourseRolloverComponent extends Component {
       },
     });
     this.changeSelectedYear(this.years.firstObject);
+    this.academicYearCrossesCalendarYearBoundaries = yield this.iliosConfig.itemFromConfig(
```

Comment:
I think this needs to move up above `allCourses` otherwise there could be a flash of wrong values while the config is loading and the yield is working.

Reply:
good catch :heavy_check_mark: